apis/externalsecrets/v1alpha1/secretstore_types.go

```diff
@@ -96,6 +96,7 @@ type SecretStoreRetrySettings struct {
 	RetryInterval *string `json:"retryInterval,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=Ready;InvalidStoreConfiguration;InvalidProviderConfig;ValidationFailed;Valid
```

Comment:
The same as https://github.com/external-secrets/external-secrets/pull/2860/files#r1385652402